images/base/entrypoint

```diff
@@ -42,6 +42,31 @@ fix_mount() {
   mount --make-rshared /
 }
 
+fix_cgroup() {
+  echo 'INFO: fix cgroup mounts for all subsystems'
+  # For each cgroup subsystem, Docker does a bind mount from the current
+  # cgroup to the root of the cgroup subsystem. For instance:
+  #   /sys/fs/cgroup/memory/docker/<cid> -> /sys/fs/cgroup/memory
+  #
+  # This will confuse Kubelet and cadvisor and will dump the following error
+  # messages in kubelet log:
+  #   `summary_sys_containers.go:47] Failed to get system container stats for ".../kubelet.service"`
```

Comment:
note that these are just cadvisor warnings, they should't affect anything?

Reply:
yes, they don't affect anything, but still annoying and spamming the kubelet logs.

Reply:
agree that removing the noise from the logs is nice, especially those kubelet ones